platform/src/main/java/org/stellar/anchor/platform/apiclient/CustodyApiClient.java

```diff
@@ -60,6 +63,20 @@ public GenerateDepositAddressResponse generateDepositAddress(String assetId)
     return gson.fromJson(responseBody, GenerateDepositAddressResponse.class);
   }
 
+  public CreateTransactionPaymentResponse createTransactionPayment(String txnId, String requestBody)
```

Comment:
We can also rename this method to `CreateTransactionPaymentResponse createPayment(String txnId, String requestBody)`